.github/workflows/ci-release.yml

```diff
@@ -50,7 +50,7 @@ jobs:
   # If this was a workflow dispatch event, we need to generate and push a tag
   # for goreleaser to grab
   version_bump:
-    needs: [lint, markdown-linter, test]
+    # needs: [lint, markdown-linter, test]
```

Comment:
```suggestion
    needs: [lint, markdown-linter, test]
```
probably?

Reply:
I did not make this change!

Reply:
This is funny :D :D

Probably wrong PR was targetted: https://github.com/celestiaorg/celestia-app/pull/2719/commits/21f244af1be156ba9891c21ff443759121b0f5ae

cc  @rootulp

Reply:
I merged a wrong branch into my branch, fixed it now!